fetchmail_notify_error_to_sender/__manifest__.py

```diff
@@ -14,7 +14,7 @@
     'license': 'AGPL-3',
     'depends': [
         'fetchmail',
-        'test_mail'
+        # 'test_mail',
```

Comment:
Why don't you just delete the line :smile:

Reply:
😑 :+1: